frontend/src/components/SignUp.jsx

```diff
@@ -50,59 +51,60 @@ function SignUp() {
-            <h2>Sign Up</h2>
-            <form onSubmit={handleSubmit}>
-              <label>
-                <h3>Name</h3>
-                <input
-                  type="text"
-                  placeholder="name"
-                  name="name"
-                  value={data.name}
-                  required
-                  onChange={handleChange}
-                />
-              </label>
-              <label>
-                <h3>Email</h3>
-                <input
-                  type="text"
-                  placeholder="email"
-                  name="email"
-                  value={data.email}
-                  required
-                  onChange={handleChange}
-                />
-              </label>
-              <label>
-                <h3>Password</h3>
-                <input
-                  type="password"
-                  placeholder="password"
-                  name="password"
-                  value={data.password}
-                  required
-                  onChange={handleChange}
-                />
-              </label>
-              <label>
-                <button type="submit">Sign Up</button>
-              </label>
-            </form>
+  render() {
+    return (
+      <div className={"SignUp.app"}>
+        <div class={SignUp.row}>
+          <div class={SignUp.column + " " + SignUp.darkPurple}>
+            <img class={SignUp.logo} src={logo} alt="Hooli" />
+          </div>
+          <div class={SignUp.column}>
+            <div class={SignUp.login}>
+              <h2>Sign Up</h2>
+              <form onSubmit={this.handleSubmit}>
+                <label>
+                  <h3>Name</h3>
+                  <input
+                    type="text"
+                    placeholder="name"
+                    name="name"
+                    value={this.state.name}
+                    required
+                    onChange={this.handleChange}
+                  />
+                </label>
+                <label>
+                  <h3>Email</h3>
+                  <input
+                    type="text"
+                    placeholder="email"
+                    name="email"
+                    value={this.state.email}
+                    required
+                    onChange={this.handleChange}
+                  />
+                </label>
+                <label>
+                  <h3>Password</h3>
+                  <input
+                    type="password"
+                    placeholder="password"
+                    name="password"
+                    value={this.state.password}
+                    required
+                    onChange={this.handleChange}
+                  />
+                </label>
+                <label>
+                  <button type="submit">Sign Up</button>
+                </label>
+              </form>
+            </div>
           </div>
         </div>
       </div>
-    </div>
-  );
+    );
+  }
 }
 
```

Comment:
at this point, add in a `export default token`

Reply:
i stored it through sessionStorage.setData(), does it accomplish the same thing?

Reply:
Yup, it's the same thing